bin/ksql-server-start

```diff
@@ -7,7 +7,7 @@
 
 if [ $# -lt 1 ]; 
 then
-    echo "USAGE: $0 [-daemon] ksql-server.properties"
+    echo "USAGE: $0 [-daemon] --config-file ksql-server.properties"
```

Comment:
Is `--config-file` mandatory ti run `ksql-server-start`? If yes this will affect the Confluent CLI too.

Reply:
hmm I'm not sure that is correct. The config file is a required argument but you don't need the `--config-file`

Reply:
Ok i see you have changed it in `ServerOptions`.

Reply:
I'm going to switch back to making this an arg, and do the same for the CLI. This is consistent with other cp platform products